src/routes/docs/products/functions/functions/+page.markdoc

```diff
@@ -0,0 +1,182 @@
+You can create functions using the CLI, without needing to access the Console.
+Before you can deploy with the Appwrite CLI, make sure you've [initialized the CLI.](/docs/tooling/command-line/installation)
+
+To deploy with the Appwrite CLI, your function must be added to your `appwrite.json` config file. 
+Use the `appwrite init function` method to create a starter function, then paste in your function code.
+
+```bash
+appwrite init function
+```
+
+The init command will initialize a folder with a starter function code.
+To deploy the generated code, add dependencies, and deploy the function using the following command:
+
+```bash
+appwrite deploy function
+```
+{% /tabsitem %}
+{% /tabs %}
+
+# Configuration {% #configuration %}
+
+## Name {% #name %}
+You can update the name of your function by navigating to your function > **Settings** > **Name**.
+Update your function's name and click **Update**.
+Functions are executed using it's ID, updating name does not affect references to your function.
+
+## Runtime {% #runtime %}
+Every deployment of a function uses the same runtime. 
+You can update the runtime of a function by navigating to function > **Settings** > **Runtime**.
+Select a new runtime and click **Update**.
+
+{% info title="Redeployment required" %}
+This change requires your function to be redeployed to take effect.
+{% /info %}
+
+When you update a function's runtime, you may need to update code of your function to make it compatible with the new runtime.
+
+## Build configuration {% #build-configuration %}
+You can update the entrypoint file and build settings of your function by navigating to your function > 
+**Settings** > **Configuration**. 
+
+{% info title="Redeployment required" %}
+This change requires your function to be redeployed to take effect.
+{% /info %}
+
+The **Entrypoint** refers to the file imported and executed by the function executor. 
+It must export a [valid function entrypoint function](/docs/products/functions/develop#entrypoint).
+It's recommended you use one of the [starter function templates](/docs/products/functions/templates) and edit from there.
+
+Under **Build settings**, you can update your build commands. 
+These are terminal commands that will be executed in the runtime containers in the build step of the deployment process.
+
+## Git integration {% #git-integration %}
+You can update the entrypoint file and build settings of your function by navigating to your function > **Settings** > **Configuration**.
+
+{% info title="Redeployment required" %}
+This change requires your function to be redeployed to take effect.
+{% /info %}
+
+Under **Git settings** you can configure the Git repository and branch that your function is connected to.
+
+Build commands and entrypoint are executed relative to the configured **Root directory** of your Git respository.
+
+By default, Appwrite will create comments in PRs to your connected branch. You can use **Silent mode** to suppress these comments.
```

Comment:
Should we explain where/how silent mode is enabled?

Reply:
The entire section, when read as a whole, talks about these Git settings together, should be fine.